test/src/unit-cppapi-subarray.cc

```diff
@@ -1174,6 +1175,7 @@ TEST_CASE(
       .set_data_buffer("a", data);
 
   // Submit query
+  subarray.set_config(cfg);
```

Comment:
Why is this necessary? The new member variable ```merge_overlapping_ranges_``` is initiallized in the constructor with a default value no?